components/RadioPlayer.vue

```diff
@@ -22,41 +30,33 @@
             <a href="#" class="text-secondary volumeshower" @mouseenter="showVolumeSlider = !showVolumeSlider">
               <i class="material-icons" aria-hidden="true">volume_down</i>
             </a>
-          </div>
 
-          <div v-show="showVolumeSlider" id="radio-player-controls" class="radio-controls-standalone volumecontrolos" @mouseleave="showVolumeSlider = !showVolumeSlider">
-            <div class="radio-control-mute-button">
-              <a href="#" class="text-secondary" :title="'mute_btn'" @click.prevent="volume = 0">
-                <i class="material-icons" aria-hidden="true">volume_mute</i>
-              </a>
-            </div>
-            <div class="radio-control-volume-slider">
-              <input
-                id="jp-volume-range"
-                v-model="volume"
-                type="range"
-                :title="'volume_slider'"
-                class="custom-range jp-volume-range"
-                min="0"
-                max="100"
-                step="1"
-              >
-            </div>
-            <div class="radio-control-max-volume-button">
-              <a href="#" class="text-secondary" :title="'full_volume_btn'" @click.prevent="volume = 100">
-                <i class="material-icons" aria-hidden="true">volume_up</i>
-              </a>
+            <div v-show="showVolumeSlider" id="radio-player-controls" class="radio-controls-standalone volumecontrolos" @mouseleave="showVolumeSlider = !showVolumeSlider">
+              <div class="radio-control-mute-button">
+                <a href="#" class="text-secondary" :title="'mute_btn'" @click.prevent="volume = 0">
```

Comment:
Does here `@click.prevent` event mean that it prevents clicking _if_ the volume is already 0? 

Great that we are adding mute functionality! This was 1 thing that people asked about a lot!!

Reply:
.prevent is preventing the browser triggering the default click event, same as event.preventDefault
mute is also possible now, it was implemented from the beginning :-)

Reply:
Yeah but not w/ click like that at least I don't remember it. Sorry if it has been there all along!